detekt-baseline.xml

```diff
@@ -233,6 +233,7 @@
     <ID>UndocumentedPublicClass:HomeMenu.kt$HomeMenu.Item$ReconnectSync : Item</ID>
     <ID>UndocumentedPublicClass:HomeMenu.kt$HomeMenu.Item$Settings : Item</ID>
     <ID>UndocumentedPublicClass:HomeMenu.kt$HomeMenu.Item$SyncAccount : Item</ID>
+    <ID>UndocumentedPublicClass:HomeMenu.kt$HomeMenu.Item$ManageAccountAndDevices : Item</ID>
```

Comment:
The baseline file should've just allowed time to add documentation but is currently abused with new ignores instead of adding the proper documentation.
Since the new menu item is new to the app and you know the context for it can you please add a small documentation for it?
I'll also open a ticket for working through the baseline files and clearing them instead of adding new ignores.